### PR TITLE
Optimization: Minimize spawn count of DummyWeaponProjectile

### PR DESCRIPTION
**Pull request by xezon**
* Follow up for #279

This change minimizes the dummy weapon firing interval and spawn count of DummyWeaponProjectile.

### CODE REVIEW

**Comment by xezon** [comment]
Tested successfully with !SkirmishTestDummyWeapon map.

File: Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini [hunk 1]
```diff
@@ -4511,13 +4522,13 @@ Weapon BattleshipBogusGun
...
+  ProjectileObject = DummyWeaponProjectile
   FireFX = WeaponFX_BattleshipBogusGun
   FireSound = BattleshipWeapon
   RadiusDamageAffects = ENEMIES NEUTRALS
-  DelayBetweenShots = 750         ; time between shots, msec
-  ClipSize = 3                    ; how many shots in a Clip (0 == infinite)
-  ClipReloadTime = 1500           ; how long to reload a Clip, msec
+  DelayBetweenShots = 999999 ; time between shots, msec
+  ClipSize = 0 ; how many shots in a Clip (0 == infinite)
+  ClipReloadTime = 0 ; how long to reload a Clip, msec
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini]: Does it still work? ClipSize sounds important.

AmericaVehicleBattleShip is used in USA02

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini]:
```ini
  Behavior = FireWeaponPower ModuleTag_FireWeapon01
    SpecialPowerTemplate = SpecialPowerBattleshipBombardment
    ;Automatically reloads weapons immediately upon trigger, then fires multiple weapon slots.
    MaxShotsToFire    = 3
  End
```
From the battle ship config. Seems like ClipSize and ClipReloadTime are used, no?

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini]: Yes looks suspicious. Changes to BattleshipBogusGun reverted.

File: Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini [hunk 2]
```diff
@@ -5878,7 +5892,7 @@ End
 ;  DamageType                  = COMANCHE_VULCAN  ;used only for this weapon so stinger sites don't lose their guys but otherwise acts just like Small_Arms
 ;  DeathType                   = NORMAL
 ;  WeaponSpeed                 = 999999.0          ; dist/sec (huge value == effectively instant)
-;  ProjectileObject            = DummyWeaponProjectile ; Patch104p @bugfix from NONE
+;  ProjectileObject            = DummyWeaponProjectile
 ;  RadiusDamageAffects         = ALLIES ENEMIES NEUTRALS
 ;  DelayBetweenShots           = 100         ; time between shots, msec
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini]:
```suggestion
;  DelayBetweenShots = 999999 ; time between shots, msec
```
We add `DummyWeaponProjectile`, may as well do this.

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/Weapon.ini]: fixed.

**Comment by xezon** [comment]
Documentation added. Looks good.

**Pull request merged**